### PR TITLE
Created tests for ImageRequestController

### PR DESCRIPTION
**Pull request by llvieira**

### CODE REVIEW

File: src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    @Autowired
+    private MockMvc mockMvc;
+
+    private ApplicationFacade facade;
+
+    @Before
+    public void setUp() throws FogbowManagerException {
+        this.facade = spy(ApplicationFacade.class);
+        PowerMockito.mockStatic(ApplicationFacade.class);
+        given(ApplicationFacade.getInstance()).willReturn(this.facade);
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java]: follow the convention. Do not use static imports. I guess there are more calls like that. Please, check this out.

File: src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        // exercise
+        MvcResult result = this.mockMvc.perform(requestBuilder).andReturn();
+
+        // verify
+        int expectedStatus = HttpStatus.OK.value();
+        assertEquals(expectedStatus, result.getResponse().getStatus());
+
+        TypeToken<Map<String, String>> token = new TypeToken<Map<String, String>>(){};
+        Map<String, String> resultMap = new Gson().fromJson(result.getResponse().getContentAsString(), token.getType());
+        assertTrue(resultMap.size() == 3);
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java]: assertEquals fits better. If there are more calls like that, please, change them.

File: src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+        // exercise
+        MvcResult result = this.mockMvc.perform(requestBuilder).andReturn();
+
+        // verify
+        int expectedStatus = HttpStatus.OK.value();
+        assertEquals(expectedStatus, result.getResponse().getStatus());
+
+        Image resultImage = new Gson().fromJson(result.getResponse().getContentAsString(), Image.class);
+        assertEquals(resultImage.getId(), image.getId());
```
Comment (thiagomanel) [inline src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java]: @llvieira I think you swapped the args. Remember to add first the expected and then the actual. Please, verify the other test cases accordingly.

File: src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java (hunk 1 above)
Comment (llvieira) [reply src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java]: Fixed!

File: src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java (hunk 2 above)
Comment (llvieira) [reply src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java]: Done!

File: src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java (hunk 3 above)
Comment (llvieira) [reply src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java]: Done!

**Pull request merged**